govtool/frontend/src/components/organisms/DashboardCards/DirectVoterDashboardCard.tsx

```diff
@@ -127,7 +127,9 @@ export const DirectVoterDashboardCard = ({
       description: (
         <Trans
           i18nKey="dashboard.cards.directVoter.registerDescription"
-          values={{ votingPower: ada }}
+          values={{
+            deposit: correctAdaFormat(voter?.deposit),
+          }}
```

Comment:
I had another look at this, so as to avoid conflicts, and turns out there was no bug, because this is a different `registerDescription` key (`"dashboard.cards.directVoter.registerDescription"`).

The one with the deposit (`"directVoter.registerDescription"`) is used [here](https://github.com/IntersectMBO/govtool/blob/develop/govtool/frontend/src/pages/RegisterAsDirectVoter.tsx#L133)

Reply:
Btw, it's the same place that fooled me, where I missed that voting power when hiding it and wrongly changed copy about the deposit 😅

Reply:
if I understand, I believe I have fixed it via [3c4b74d](https://github.com/IntersectMBO/govtool/pull/2005/commits/3c4b74da0e4263cb24242948a80806e37c552737)